components/quickbooks/actions/sandbox-search-invoices/sandbox-search-invoices.mjs

```diff
@@ -1,84 +1,86 @@
-// legacy_hash_id: a_OOiLgK
-import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+import quickbooks from "../../quickbooks.app.mjs";
 
 export default {
   key: "quickbooks-sandbox-search-invoices",
   name: "Search Invoices",
-  description: "Searches for invoices.",
-  version: "0.1.1",
+  description: "Searches for invoices. [See docs here](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/invoice#query-an-invoice)",
+  version: "0.1.2",
   type: "action",
   props: {
-    quickbooks: {
-      type: "app",
-      app: "quickbooks",
-    },
-    include_clause: {
-      type: "string",
+    quickbooks,
+    includeClause: {
       description: "Fields to use in the select clause of the Invoice data query. See query language syntax, limitations, and other specifications on [Data queries](https://developer.intuit.com/app/developer/qbo/docs/develop/explore-the-quickbooks-online-api/data-queries)",
-    },
-    where_clause: {
+      label: "Include Clause",
       type: "string",
-      description: "Filters to use in the where clause of the Invoice data query. Note: Multiple clauses (filters) are AND'd. The OR operation is not supported.",
     },
-    order_clause: {
-      type: "string",
-      description: "The `order_clause` is for sorting the result. Include the Invoice property to sort by. The default sort order is ascending; to indicate descending sort order, include DESC, for example: `Name DESC`",
+    maxResults: {
+      description: "The number of Invoice entity elements in the response.",
+      label: "Max Results",
       optional: true,
+      type: "string",
     },
-    start_position: {
+    orderClause: {
+      description: "The `orderClause` is for sorting the result. Include the Invoice property to sort by. The default sort order is ascending; to indicate descending sort order, include DESC, for example: `Name DESC`",
+      label: "Order Clause",
+      optional: true,
       type: "string",
+    },
+    startPosition: {
```

Comment:
Can we move `startPosition` to the app file? It is used in all querying actions.